### PR TITLE
[zero] added error message to handle on-the-fly import of torch Module class

### PR DESCRIPTION
**Pull request by FrankLeeeee**
The zero init context has a precondition that the module class has to be imported before entering the context. However, sometimes this is not always so. For example:

```
with ctx:
    model = transformers.BertModel(...)
```

This example will raise exception because BertModel is only imported on the fly. This PR adds an error message for easier debugging.

### CODE REVIEW

File: colossalai/utils/model/utils.py [hunk 1]
```diff
@@ -80,6 +80,10 @@ def __exit__(self, exc_type, exc_value, traceback):
             torch.set_default_dtype(self._old_default_dtype)
 
         def _disable_class(cls):
+            if not hasattr(cls, '_old_init'):
+                raise AttributeError(
+                    f"_old_init is not found in the {cls.__name__}, please make sure that you have this class imported before entering the context."
```
Comment (feifeibear) [inline colossalai/utils/model/utils.py:85]: make sure import InsertPostInitMethodToModuleSubClasses?

Comment (FrankLeeeee) [reply colossalai/utils/model/utils.py:85]: make sure cls is imported

Comment (FrankLeeeee) [reply colossalai/utils/model/utils.py:85]: I can make it clearer

Comment (FrankLeeeee) [reply colossalai/utils/model/utils.py]: done

**Review by feifeibear — APPROVED** [review]

**Pull request merged**